python/rgbmatrix/cppinc.pxd

```diff
@@ -40,6 +40,37 @@ cdef extern from "led-matrix.h" namespace "rgb_matrix":
         bool SetPWMBits(uint8_t)
         uint8_t pwmbits()
 
+    struct RuntimeOptions:
+      RuntimeOptions() except +
+      int gpio_slowdown
+      int daemon
+      int drop_privileges
+
+
+    RGBMatrix *CreateMatrixFromOptions(Options &options, RuntimeOptions runtime_options)
+
+
+
+cdef extern from "led-matrix.h" namespace "rgb_matrix::RGBMatrix":
+    cdef struct Options:
```

Comment:
Re ordering is not a problem. Adding or removing options to the Options struct in C++ requires adding/removing them to the Python definition and recompiling the Python library.